### DISCUSSION TITLE
Not able to install

### DISCUSSION THREAD

**Discussion by KuyaAlfie** [teslamate-org/teslamate#2143] [Q&A]
I'm a beginner and I'm trying to install teslamate.  I already have Grafana running, monitoring my docker containers and system metrics.  When trying to install Teslamate, I'm running into this error:

fana_1 (75e96d2d64b25d611a735ed1ef2ffdacac8d7139b9e168613c80b0d572de7d6c): Bind for 0.0.0.0:3000 failed: port is already allocated

ERROR: for grafana  Cannot start service grafana: driver failed programming external connectivity on endpoint teslamate_grafana_1 (75e9             6d2d64b25d611a735ed1ef2ffdacac8d7139b9e168613c80b0d572de7d6c): Bind for 0.0.0.0:3000 failed: port is already allocated
ERROR: Encountered errors while bringing up the project.

I have a container, "monitoring-grafana", running when I try to perform the install.  Is this because I have Garafa currently running at the time of the install?  

![image](https://user-images.githubusercontent.com/39384222/141351129-b4db04cc-e8b7-4fda-ad46-0b6929f245a8.png)

Any assistance would be appreciated.

Thank you!

Al

**Comment by eliluong** [comment]
It's probably because the monitoring-grafana is using port 3000, and the teslamate grafana is trying to use the same port when it starts but it cannot.

**Reply by KuyaAlfie** [reply]
Yes, I figured that out.  I used port 3001.  After doing so, Grafana saw it but I wasn't able to access login screen at port 4000. Not so easy install.
